code/weapon/trails.cpp

```diff
@@ -129,6 +129,12 @@ void trail_render( trail * trailp )
 		return;
 
 	trail_info *ti	= &trailp->info;
+
+	if (ti->texture.bitmap_id <= 0) {
+		Warning(LOCATION, "Trail bitmap %s could not be loaded.", ti->texture.filename);
```

Comment:
One last thing, if you'll permit me, the warning in `weapon_load_bitmaps` *should* have already warned about this, so we shouldn't warn again here, especially since in a render function it would be triggering every frame which is always very annoying.

Reply:
Good? Honestly, if you clicked through the first warning, getting annoyed at the others is on you.

Reply:
There's no reason to repeatedly bother the user when they've knowingly accepted and ignored the issue. There are a host of legitimate reasons why someone may understand a warning and proceed anyway. They may be attempting to fix a different issue. They may not be the actual modder, but instead someone else trying to verify some other aspect of the mod.

Reply:
fair enough